db/db_test2.cc

```diff
@@ -86,6 +86,51 @@ TEST_F(DBTest2, OpenForReadOnlyWithColumnFamilies) {
   // With create_if_missing false, there should not be a dir in the file system
   ASSERT_NOK(env_->FileExists(dbname));
 }
+
+TEST_F(DBTest2, ReadOnlyWithCompresedCache) {
+  ASSERT_OK(Put("foo", "bar"));
+  ASSERT_OK(Put("foo2", "barbarbarbarbarbarbarbar"));
+  ASSERT_OK(Flush());
+
+  for (int max_open_files : {-1, 100}) {
+    for (bool use_mmap : {false, true}) {
```

Comment:
I realize this is only four combinations (at least for now :)) but ideally this would be a parameterized test case.

Reply:
OK. Will do it.